### PR TITLE
Added play-button ui and animation

### PR DESCRIPTION
**Pull request by mouakiang**
Added play button ui and animation

### CODE REVIEW

File: src/routes/player.tsx [hunk 1]
```diff
@@ -36,16 +35,13 @@ export default function Player() {
           className={`w-60 h-60 bg-alabaster-100 border-[1px] border-black shadow-md rounded-full self-center`}
           variants={itemVariants}
         ></motion.div>
-        <div className={`flex flex-row justify-around`}>
-          <div className={``}>
+        <div className={`flex flex-row justify-center items-center`}>
```
Comment (dizzyjaguar) [inline src/routes/player.tsx]: We shouldn't use `mr-60`. Instead there should be a parent container that holds 2 containers that are spaced apart. The **green square** being the parent container. Thats what the `justify-around` was being used for previously. The **blue squares** being the child containers, which in the case of the 3 action buttons, can be used to control spacing with `spacing-x-{whatEverNumber} _instead_ of `mr-{number}`.
Screenshot 2024-05-04 at 1 48 20 PM

Comment (dizzyjaguar) [reply src/routes/player.tsx]: **Also**,
In terms of commit name, there is no need to include the word `Tailwind`. A better commit message would be _Update record and play button styling_ 😄

File: package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "jest": "^29.7.0",
     "postcss": "^8.4.27",
     "prettier": "3.0.0",
-    "tailwindcss": "^3.3.3",
+    "tailwindcss": "^3.4.3",
```
Comment (dizzyjaguar) [inline package.json:37]: Keep any package updates as a separate commit. **Example**: _Update tailwind_

**Review by dizzyjaguar — CHANGES_REQUESTED** [review]
Lets change the animation for the `play button`. For `play` instead of having it rotate, lets have it just ever so slightly move in the direction that the arrow points (to the right). 
A few nits here and there but overall it looks good 😁

File: src/routes/player.tsx (hunk 1 above)
Comment (mouakiang) [reply src/routes/player.tsx]: So, I made the changes per your comment above. I am coming across an issue with the spacing-x-number. Even though I placed that in the child component, it seems that nothing is happening on the localhost server. Example: 
>div className={`spacing-x-10`}<
          >RecordButton /<
          >/div<
          
 Record button on localserver shows no changes. 
 *Don't mind the backwards arrows lol that's not how it is in the code.

File: package.json (hunk 2 above)
Comment (mouakiang) [reply package.json:37]: Duly noted, thank you!

File: src/routes/player.tsx (hunk 1 above)
Comment (dizzyjaguar) [reply src/routes/player.tsx]: @mouakiang  can you post a fuller snippet of this? Of the div and the child comps

Comment (mouakiang) [reply src/routes/player.tsx:38]: Code Snippet

Comment (mouakiang) [reply src/routes/player.tsx]: snippet
